### ISSUE TITLE
Improving how Dask looks on the web

### ISSUE THREAD

**Issue by jacobtomlinson** [dask/community#220]
Hey all. I just wanted to provide an update on the progress made by the working group thinking about Dask's branding, websites, visual identity, etc. We will add more updates to this issue as things progress.

Coiled has provided some design resources for taking a high level look at Dask's websites, logos, colour schemes, etc. To make the best use of this resource @adireske, @jsignell, @mrocklin, @jrbourbeau and I have been meeting with Adi's team to talk about how we can improve things.

This effort started with a look at how Dask's various websites are performing in terms of getting the right information in front of folks. We have a bunch of different "foo.dask.org" Sphinx sites for the various subprojects under Dask. We also have a landing page at dask.org, a blog at blog.dask.org and the Dask dashboard. All of these web pages exist for different reason, are built in different ways and are often maintained by different folks. One big goal of the working group is to try and improve how Dask performs on search engines when folks are looking for ways to scale Pandas, NumPy and the PyData ecosystem. The way things are today aren't doing us many favours. We had an expert deliver some updates at the last Dask Community meeting which you can find summarised in [these slides](https://docs.google.com/presentation/d/1mWUXLPzZqw2Ttq2V3-UD6WUesw_5Zjad7lCaFGsKAOQ/edit?usp=sharing).

<img width="2547" alt="image" src="https://user-images.githubusercontent.com/1610850/154102233-54b437e1-b876-463a-bc50-946339551ae1.png">

To improve this we are looking at building a new website for Dask which will hold non-documentation content as well as refreshing the documentation as a whole (#170). 

The other task we've been working on is thinking about Dask's branding. This was started in #135 and #140 where we looked into things like colour schemes, diagrams, charts, etc.

The working group has rebooted those efforts as part of the holistic look at Dask's identity on the web and we are now exploring things like a logo refresh, colour pallettes, etc. Ideally we want to get to a point where the designers can give us enough structure that those of us (including me) who work on things like the Dask dashboard can apply colours and layouts that fit with the rest of Dask.

The first step on this activity has been to explore updating the Dask logo and I wanted to share some options that we are considering here. Ultimately the decision will be down to the working group but feedback is always welcome. These logos represent incremental change, evolutionary change and revolutionary change. You can read more about the [thought process behind this here](https://www.dropbox.com/s/oux6ixgmf9arht0/Dask%20Brand%20Delivery-round%202.pdf?dl=1).

<img width="1301" alt="image" src="https://user-images.githubusercontent.com/1610850/154101862-0715bb4e-41e3-4ae2-ace0-bcd5b151a905.png">
<img width="1372" alt="image" src="https://user-images.githubusercontent.com/1610850/154101936-a693666f-3252-48eb-88a6-6434e6155e95.png">

Once we've made some decisions here we will move on to addressing the goals of #135 where we can explore colours that we can use throughout Dask and iconography that members of the Dask community can use to build new diagrams that fit with our style.

**Comment by ncclementi** [comment]
Thanks for putting this up @jacobtomlinson, I'm adding here some extra options that were brought up in one of the meetings. 

- For the case of the Parallel logo, make it fit in a hexagon shape.
- Other possible combinations 

![Screen Shot 2022-02-15 at 2 26 23 PM](https://user-images.githubusercontent.com/7526622/154134391-afe4c4d3-cfdb-42d7-93e1-e2ebdfc68d70.png)

![Screen Shot 2022-02-15 at 2 24 16 PM](https://user-images.githubusercontent.com/7526622/154134021-a89a4444-6e24-4e97-80fa-20b29d7edef3.png)

![Screen Shot 2022-02-15 at 2 24 37 PM](https://user-images.githubusercontent.com/7526622/154134071-b6151e0b-1a3d-488a-b605-e90ff1c1f26d.png)

**Comment by hayesgb** [comment]
I really like the parallel computing logo.  I have mixed feelings on the lowercase vs uppercase.

From the recording, there was some interesting conversation about upper vs lowercase. I have a preference for the upper case.  Thoughts?

**Comment by jcrist** [comment]
One of my biggest pet peeves is people that write dask in all caps, so my preference is for the lowercase (or title case like "Dask").

**Comment by mrocklin** [comment]
I share the pet peave about people upper-casing DASK in communication, however I think that we shouldn't use the logo in order to solve that problem.  That feels like the tail wagging the dog a little.  Aesthetically I like the upper-case.  The letters fill out the space a bit better to my eye.  That's totally subjective though.

**Comment by jsignell** [comment]
I tend to agree with Jim that the logo being all caps leads people to think it's an acronym. This is definitely subjective, but I also think the lowercase feels more approachable and friendly.

I have been thinking about it and I really do like the simplified/single color wings. If it's not too much, I was wondering if we could see an all lower-case dask with the single color wings? 

In my mind some of the good things about the wings are:
1) They don't look like any other logos
2) They work really well in single color on tshirts with dark or light backgrounds
3) The color goes well with a black "dask" word for cases where the style of the word is out of our control (I'm mostly thinking of github)
4) People already associate the logo with Dask. (I know this is kind of unfair to the other logos, but it is a real pro)

**Comment by jacobtomlinson** [comment]
I also think any of the options here are an improvement over what we have today. This is exciting!

**Comment by martindurant** [comment]
I would also vote for keeping the sails/wings, sorry. I find the rectangles generic, especially in the coding world. I think having something does that _not_ fit a typical hexagon is a good thing!

**Comment by mrocklin** [comment]
I like both the evolution and parallel computing logos.  I think that Dask could use a more modern image.  I also want to give a +1 to this comment from @jacobtomlinson 

> It is fresh and represents a new chapter for Dask

I'd like for us to think about Dask as dynamic and changing, and not as static with past decisions.  This applies to technology choices (I've been urging people to revisit old decisions/constraints) and I think that it should apply visually as well.  We're all going to have a bias for the familiar.  I think that we should prioritize not for existing users, but for future users who aren't yet bought in to the old design.

**Comment by mrocklin** [comment]
> It feels like most people are happier with the lower case option and the rectangles icon

FWIW I'm seeing two in favor of lower case, two in favor of upper case, and one asking to see title case.

**Comment by mrocklin** [comment]
You're right, I shouldn't have said "all".  But in general people have a strong bias for the familiar.  

Changing logos has a long history of pretty much everyone not liking the change because it's familiar, and then once it happens people connect pretty strongly to the new logo.

So yeah, I didn't mean to say anything about folks here in particular, just about humans and this process in general.

**Comment by ian-r-rose** [comment]
My two cents: I like the "evolution" logo. It rhymes with the older logo, but does look fresher. I also agree with @jsignell that the lower-case text looks friendliest and is least likely to be confused for an acronym.

The "parallel squares" looks a touch generic to me, and is perhaps a bit too similar to xarray: 
![image](https://user-images.githubusercontent.com/5728311/154582097-14b36263-ad00-41ee-9865-020d16804585.png)

**Comment by mrocklin** [comment]
If folks like podcasts, I recommend listening to this 99 Percent Invisible podcast on making marks and logos

https://99percentinvisible.org/episode/making-mark-visual-identity-tom-geismar/

**Comment by mrocklin** [comment]
Oh wait, no, I think that it was this episode: https://99percentinvisible.org/episode/negative-space-logo-design-michael-bierut/

**Comment by mrocklin** [comment]
I just showed the slide decks to a friend who is not in our space for neutral feedback.  General takeaways:

- Doesn't like the combination of the uppercase DASK with the block font, specifically when combined with the rectangles logo because it feels too industrial.  "Looks like Maersk or some other large logistics mega-company"
- Likes the wings, "feels tech and modern" especially with the newer futura font combination
- Rectangles seems approachable.  Wings seems "elevated" / "silicon valley looking"
- Evolutionary logo looks like "retail tech", interpret it as "paper flipping"
- Curious about lower-case with the futura font and with other logo options

Don't ask me to interpret most of that (I don't know what paper flipping means :) ). I did find it interesting to ask people outside of our space their opinion.  I wouldn't over-rotate on these views.  I do recommend that folks present this presentation to other folks to get their opinion.  It's a fun ride 😄

**Comment by jakirkham** [comment]
Agree with what Ian said above ( https://github.com/dask/community/issues/220#issuecomment-1043549109 ).

Evolution is a nice touch up.

Too many things in the ecosystem with squares/cubes/etc.. Would rather skip this.

Also ok with wings (either as-is now or touched up).

Have we taken into account color blindness when deciding colors?

**Comment by GenevieveBuckley** [comment]
Very cool! 

I like both Wings and Evolution a lot. I really like the colour scheme in Evolution, that's a nice one. I don't feel any different emotions based on which way "up" the shapes go, although perhaps other people might.

I don't like Parallel. The shapes feel very generic to me, like they could apply to many different companies. The shapes in the other options feel more dynamic to me, and more recognizably unique for branding purposes.

It's always fun to see parts of what the designers are doing, thanks for sharing a peek into the process.

**Comment by hayesgb** [comment]
At the start of the conversation, I was a little unsure of the switch to lower case, but I really like this coupled with the new logo.

**Comment by mrocklin** [comment]
Just so that people are aware of timing constraints, I think that the web team is planning to move forward with this.  If folks have thoughts on these deisgns that they'd like considered, now is a much better time to bring up these thoughts than later down the line.

**Comment by AdiReske** [comment]
Hi, 

On Monday next week (the 6th) we are finally going live with the new Dask website and visual look. We are also refreshing the Youtube look to match as well as the docs and other channels. 

As you may or may not know, a new website is a little bit like a new beta launch. We may need some hot fixes in the first few days. Things may break or appear in the wrong place. Please be patient with us as we are trying to make this transition as smooth as possible. 

Here is a link to the draft site https://dask-2-0.webflow.io

As you are reviewing, I'd like you all to look for major structural issues that would be roadblocks to going live. By this I mean core content missing, or links improperly working. We currently have mobile responsive comments noted on an internal list as well as logo updates for the home page. 

Blogs: We’re working on integrating the current posts & will have them included before going live. These are placeholders. 

Brand Guide: is still being build for mobile devices. 

I also want to share that this website is V1 and just like any other software there will be sprints and V2 and V3 etc. We hope that makes sense.

**Comment by MrPowers** [comment]
Site looks great, messaging is on point.

Perhaps we can update the page title from DASK => Dask, so it matches the logo.

<img width="359" alt="Screen Shot 2022-06-01 at 4 09 09 PM" src="https://user-images.githubusercontent.com/2722395/171502393-2a1eaa48-7e05-4e63-9812-34c69dac7c76.png">

Great work!

**Comment by jakirkham** [comment]
Seeing "Community" twice in the header. One appears to show a dropdown menu. Other doesn't have a link. Guessing one of these should be dropped?

Also on Safari 15.3 on macOS 12.2.1.

<img width="1792" alt="Screen Shot 2022-06-01 at 6 09 14 PM" src="https://user-images.githubusercontent.com/3019665/171526269-ab994d49-f942-4751-83b0-36931b6d3771.png">

**Comment by pavithraes** [comment]
Thanks for sharing, @AdiReske! The site looks great!

Some notes:
* In https://dask-2-0.webflow.io/
    * At the very end of the page, the `conda install dask` gif is covering the footer for me, and
    * some links in the footer seem to not be connected, specifically: all social icons, privacy policy, and terms of service   
     <img width="1439" alt="Screenshot 2022-06-02 at 6 17 36 PM" src="https://user-images.githubusercontent.com/33131404/171632611-53fca6d9-ff03-4d8a-b9ce-86fcbdbc7b5d.png">
* In https://dask-2-0.webflow.io/get-started,
    * I think the description for "Dask Tutorial" needs to be updated? 
    ![Screenshot 2022-06-02 at 5 48 12 PM](https://user-images.githubusercontent.com/33131404/171627829-802a736c-d475-4cf9-9691-5408ab2c89d6.png)
* In https://dask-2-0.webflow.io/news-events (I believe this is still being built), 
  * I couldn't close the "Subscribe to the calendar" pop up by clicking on the cross here:
    ![Screenshot 2022-06-02 at 6 01 30 PM](https://user-images.githubusercontent.com/33131404/171630222-b4239a93-2045-44f0-be79-d23b04b89218.png)
* Minor comment -- In https://dask-2-0.webflow.io/get-help,
   * Under "create a minimal example", the "see also" link isn't connected:
    ![Screenshot 2022-06-02 at 5 56 16 PM](https://user-images.githubusercontent.com/33131404/171628865-47008041-9433-4456-a65e-12b0d58c066d.png)

I'm on Chrome 102.0.5005.61 (arm64), macOS 12.1

**Comment by fjetter** [comment]
Is it intentional that there is no shortcut / link to the documentation on the landing page? I eventually found a link at the bottom of the "Get Started" page but that was very hard to find. I would've expected this in the header next to Community/Blog/Get Started

**Comment by ian-r-rose** [comment]
Quick note: the animation shows `"FILE.IPYMB"`, and doesn't appear to be a notebook :)

I'd recommend calling it something like `WORKFLOW.PY`

**Comment by shughes-uk** [comment]
There's on docs link directly in the header bar. I'm frequently annoyed by products that don't have prominent docs links on their SEOed landing pages, please please add one and make it super obvious!

**Comment by AdiReske** [comment]
good call about the link to docs. Thank you!

**Comment by pavithraes** [comment]
> @pavithraes - can you please send me a better description of the training you referred to?

@AdiReske Sure!

The ["Dask Tutorial" card](https://dask-2-0.webflow.io/get-started) has the following description currently: "Open-Source machine learning for time series analysis". However, it links to https://tutorial.dask.org/, which covers a lot more than machine learning and time series analysis -- it is a complete introductory tutorial presented at SciPy that talks about all collections, the distributed scheduler, and more.

The description could instead be something like:
> This collection of Jupyter Notebooks, presented in the Dask Tutorial at SciPy, helps new users get started with Dask.

**Comment by hendrikmakait** [comment]
This looks awesome! I have one usability issue: The hover effect of the elements in [https://dask-2-0.webflow.io/#Advantages](https://dask-2-0.webflow.io/#Advantages) has confused me quite a bit. So far all color-changing elements on the website were clickable, but these are not even though they have one of the most visible hover effects of any element I was able to find. The only contenders are the individual blog posts at [https://dask-2-0.webflow.io/blog](https://dask-2-0.webflow.io/blog), which are clickable. Is there a plan to link these to some other piece of documentation? Not that the elements above under **Why Choose Dask?** are also "descriptive boxes", yet they do not have any interactive effect attached to them.

**Comment by hendrikmakait** [comment]
Other small things I noticed:
* I find it relatively hard to go from that page to our Github. Should we feature Github under **Community** or even as a hard link in the top navigation? I often just very quickly skim some project's landing page and then click through to their Github. On our page, this is close to impossible. [Ray](https://www.ray.io/) on the other hand takes this to the extreme with the feature in the top bar. 
* The top navigation does not stick. Should it? I guess I would've expected it to do so since it would make it easier to navigate to other pages without having to scroll to the top again. Then again, this might look awkward with the sidebar.
* We have a **Contribute** link under **Blog**, but not under **Community**. Should we feature a more prominent link to the resources on actively contributing to Dask?

**Comment by AdiReske** [comment]
Thank you @pavithraes !

@hendrikmakait these are all good suggestions that we will add to the v2 of the website. Thank you!!

**Comment by ntabris** [comment]
The Coiled logo image used on the new website is a low resolution png

![image](https://user-images.githubusercontent.com/46289310/171877349-0394c3b4-a343-46b3-aaf5-a12537207955.png)

I see this both on https://dask-2-0.webflow.io/get-started and https://dask-2-0.webflow.io/powered-by

The current dask website uses a much better image

![image](https://user-images.githubusercontent.com/46289310/171878585-6b37aa69-3d9d-4713-80cf-87ccd7fa9e3e.png)

**Comment by ntabris** [comment]
The "mailing list" link in "Community" menu at top brings up the overlay card but doesn't take you to "mailing list" tab of the card if you weren't already there. The "mailing list" link in footer does work correctly.

**Comment by AdiReske** [comment]
We just went live with the new site. Still working on the blogs section and pushing the rebranded docs.

**Comment by douglasdavis** [comment]
> Quick note: the animation shows `"FILE.IPYMB"`, and doesn't appear to be a notebook :)
> 
> I'd recommend calling it something like `WORKFLOW.PY`

re-pinging Ian's comment ^^^

**Comment by jacobtomlinson** [comment]
@douglasdavis @ian-r-rose fixed now :)

**Comment by gjoseph92** [comment]
Meta-comment: this thread is collecting ideas for things to fix, discussion around them, and then confirmation that they've been fixed.

Most other places in the dask world, that would accomplished through GitHub issues per idea, PRs, and discussion on both. Previously, that would have occurred in the https://github.com/dask/dask.github.io repo (which I imagine should be archived now).

1. How will the dask community have ownership of the content on dask.org now? As far as I understand, it's now effectively closed-source / off GitHub since it's built in Webflow. A few people have access to modify it. When other people see something wrong, how are they empowered to fix it (as they would be with anything else in open-source dask?)
2. How will we track issues and fixes now? i.e., what's the equivalent of the public issue -> PR -> code review -> merge flow now? Should we have an issue-only GitHub repo for now at least, so we don't just keep adding to this thread (which would be hard for anyone to find who didn't know about it)?

**Comment by jacobtomlinson** [comment]
@gjoseph92 the plan is to move from this issue to a mailing list where website-related things can be tracked. Previously everything was very developer-centric, part of the goal here is to become more inclusive towards non-developers. For example contributing blog posts is going to shift from needing to raise a PR to a Jekyll blog to needing to submit a Google Doc to a public mailing list.

We are definitely making trade-offs with this change. The main exchange here is increasing friction for a few (those of us who will have to maintain this new system) in order to reduce friction for many.

The goal is still to be as transparent as possible. Coordination will be done in public groups, and updating of the website and content will be managed by the web steering group, which effectively anyone can join in exchange for a long-term commitment to help manage it.

**Comment by TomAugspurger** [comment]
Question about the new dask-sphinx-theme. On https://ml.dask.org/ I get a double vertical scrollbar:

<img width="1281" alt="image" src="https://user-images.githubusercontent.com/1312546/172217360-00f68d85-09d3-4913-9685-851017dd8848.png">

And I'm not sure if it's related, but the footer is cutoff halfway:

<img width="1152" alt="image" src="https://user-images.githubusercontent.com/1312546/172217461-d47789bf-2ab9-41ce-84c1-6a4fe144bfdc.png">

**Comment by gjoseph92** [comment]
> the plan is to move from this issue to a mailing list.

This seems a bit harder to manage to me, and easy for things to get lost. (See this thread, for reference.) What's wrong with an issue-only GitHub repo (just like `dask/community`)? Why create a new communication pattern here when the one we use everywhere else seems to work pretty well?

**Comment by gjoseph92** [comment]
In the meantime, I'll keep adding to this thread, since that seems to be the approach for now:

Landing page
1. I land on the homepage and after reading every section, I still don't know what dask _is_. If I've never head of dask before, I'm aware that it claims to be able to "scale any Python code" and "deploy anywhere"... but what is it that I'm deploying? I would have expected a one-sentence explanation underneath "Scale the Python tools you love". Like "Dask is a flexible library for parallel computing in Python", as the sub-heading on the docs landing page explains.
1. What’s the motivation for the code samples being animated? They’re so fast, I can barely read them, and I know dask. If I wasn’t that familiar with it, they wouldn't be meaningful to me. Plus, they’re not copy-pasteable. A static code snippet seems much more useful to me, since you can read, understand, and copy it at your own pace.
2. Why are the installation instructions at the very bottom of the page, and not copy-pasteable, but animated?
3. "Scale PyData libraries" links only to "learn about DataFrames", but also mentions arrays and scikit-learn. We should arguably link to all three.
4. "Scale any Python code" links to Futures. I'd say Delayed is a better and more commonly-used and approachable interface for doing this.

Get started page
1. The copy
    > Dask is included by default in Anaconda. You can also install Dask with Pip, or you have several options for installing from source. You can also use Conda to update Dask or to do a minimal Dask install.

    is somewhat awkward and verbose for explaining a very simple thing. Installing from source is not relevant to most people; the fact that you can "use conda to update dask" makes it sound like this is some special dask feature as opposed to just what any package manger does. Why not just "you can install dask with pip or conda"?
2. "Learn your way around" suggests links to tutorials and videos, but doesn't mention the docs.

**Comment by ayushdg** [comment]
> Question about the new dask-sphinx-theme. On https://ml.dask.org/ I get a double vertical scrollbar

> And I'm not sure if it's related, but the footer is cutoff halfway

I see the same issue on the dask-sql doc page as well:
![image](https://user-images.githubusercontent.com/19949207/172249604-7627c7d5-5bfd-46f9-9a62-df69ebf47bd7.png)

**Comment by jrbourbeau** [comment]
Thanks @TomAugspurger @ayushdg -- I opened a separate issue for the sphinx theme over in the `dask/dask-sphinx-theme` repo https://github.com/dask/dask-sphinx-theme/issues/74

**Comment by mrocklin** [comment]
Amidst all of the micro-recommendations (thanks for the attention all!) I also wanted to take a moment and give a big shoutout to folks for getting this done.  In particular I want to call out @jsignell @jacobtomlinson and @AdiReske for pushing this process through.  The new style looks great, and I'm really impressed that we pulled everything off consistently and without hiccups.  

Thank you everyone who contributed here!

**Comment by jakirkham** [comment]
Have we shared the new logo with NumFOCUS? Looks like they are still using [the old one]( https://numfocus.org/project/dask )

Also may want to sync with them about stickers (so they are using the new logo as well)

**Comment by AdiReske** [comment]
we did https://numfocus.myspreadshop.com/all but good call! i'll ask them to update this page as well @jakirkham

**Comment by jakirkham** [comment]
Thanks @AdiReske! 🙏

May also be worth mentioning using this on stickers or other swag. SciPy 2022 is coming up soon and the NumFOCUS Summit in September. So it would be good to make sure anything they have there is using the new logo. Sounds like you are already engaged with NumFOCUS, but happy to help connect you if that would help 🙂

**Comment by AdiReske** [comment]
yes! here is he sticker and I'll ask Jim to bring it with them https://numfocus.myspreadshop.com/dask+logo+sticker?idea=629e25e254e3e23bfd511fea  we also have https://www.pythonatscale.com/ in October with Numfocus

**Comment by rrpelgrim** [comment]
Thanks @jakirkham and @AdiReske. At PyData London last weekend Numfocus still had the old logo up. I pointed this out to Jim and he assured me the new logo would be used going forward.
